src/main/java/xaeroplus/settings/XaeroPlusSettingRegistry.java

```diff
@@ -90,12 +93,18 @@ public final class XaeroPlusSettingRegistry {
     public static final XaeroPlusSetting owAutoWaypointDimension = XaeroPlusSetting.createBooleanSetting("Prefer Overworld Waypoints",
                     "Prefer creating and viewing Overworld waypoints when in the nether.",
             true);
-
     public static final XaeroPlusSetting showWaypointDistances = XaeroPlusSetting.createBooleanSetting("Show Waypoint Distances",
             "Display the distance to every waypoint in the full waypoint menu.",
             true);
+    public static final XaeroPlusSetting showRenderDistanceSetting = XaeroPlusSetting.createBooleanSetting("Show Render Distance",
+            "Show server side render distance (actually just another setting)",
+            false);
+    public static final XaeroPlusSetting assumedServerRenderDistanceSetting = XaeroPlusSetting.createFloatSetting("Server Render Distance",
+            1f, 32f, 1f,
+            "width of the square for displaying the render distance",
+            9f); // 2b2t currently returns a 9 wide square to the client
```

Comment:
The `view-distance` server config has a separate scale from the client's `Render Distance` setting.

In order to match the two we can use following equation: `Render Distance` = (`view-distance` * 2) + 1

Users will generally be more familiar with the config on their side but this equation means there are lots of numbers that could be input to this setting that don't actually align with a possible `view-distance` state.

I think we should scale this in `view-distance` terms. e.g. 2b2t's `view-distance` is 4.